src/app/routes/api/gmail.py

```diff
@@ -0,0 +1,134 @@
+"""
+This file contains the GMAIL API authorization and automated email processing for 
+the Service Account
+"""
+from __future__ import print_function
+from googleapiclient.discovery import build
+from apiclient import errors
+from email.mime.text import MIMEText
+import pybase64
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+
+
+# Email sending confirmation email
+EMAIL_FROM = "admin@piecemealapp2.com"
+
+
+def confirmation_email(new_user_email):
+    """
+
+    Initiates the issuance of account creation confirmation email
+
+    Args:
+        New user email address provided during signup process
+
+    """
+
+    # Email from user must be retrieved and stored in EMAIL_TO
+    EMAIL_TO = new_user_email
+    EMAIL_SUBJECT = "Welcome to pieceMeal!"
+    EMAIL_CONTENT = "Welcome to pieceMeal! \nThank you for creating a new account with us. \nPlease sign in at https://www.piecemealapp2.com to complete your profile"
```

Comment:
This line is too long. Make sure you run the linter (PyLint) to see if you have any linting errors